### ISSUE TITLE
ossec-makelists is failling on CentOS 8

### ISSUE THREAD

**Issue by xr09** [wazuh/wazuh-ansible#338]
### Description

There's an error while deploying `3.11.0_7.5.1` to a CentOS 8 host, more specifically when building cdb_lists.

```
RUNNING HANDLER [../roles/wazuh/ansible-wazuh-manager : rebuild cdb_lists] *************************************
Friday 27 December 2019  10:50:38 +0100 (0:00:00.026)       0:01:31.151 ******* 
fatal: [centos8]: FAILED! => {"changed": true, "cmd": ["/var/ossec/bin/ossec-makelists"], "delta": "0:00:00.003934", "end": "2019-12-27 09:50:39.175751", "msg": "non-zero return code", "rc": 1, "start": "2019-12-27 09:50:39.171817", "stderr": "2019/12/27 09:50:39 ossec-testrule: CRITICAL: (1226): Error reading XML file '/var/ossec/etc/ossec.conf':  (line 0).", "stderr_lines": ["2019/12/27 09:50:39 ossec-testrule: CRITICAL: (1226): Error reading XML file '/var/ossec/etc/ossec.conf':  (line 0)."], "stdout": "", "stdout_lines": []}
```

### Tasks
- [x] Research more info about this on Core 
- [x] Fix any formatting errors on the XML file

**Comment by xr09** [comment]
Bug origin traced to #279

**Comment by xr09** [comment]
Fixed with #340
